src/views/TechnicalAssistance/AdminHome/Consult.tsx

```diff
@@ -0,0 +1,299 @@
+
+  const [mutate] = useMutation<
+    UpdateTrbRequestConsultMeeting,
+    UpdateTrbRequestConsultMeetingVariables
+  >(UpdateTrbRequestConsultMeetingQuery);
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors, isDirty, isSubmitting }
+  } = useForm({
+    defaultValues: {
+      meetingDate: '',
+      meetingTime: '',
+      notes: ''
+    }
+  });
+
+  const hasErrors = Object.keys(errors).length > 0;
+
+  useEffect(() => {
+    if (hasErrors) {
+      const err = document.querySelector('.trb-basic-fields-error');
+      err?.scrollIntoView();
+    }
+  }, [errors, hasErrors]);
+
+  return (
+    <GridContainer className="width-full">
+      <Breadcrumbs
+        items={[
+          { text: t('Home'), url: `/trb` },
+          {
+            text: t('adminHome.breadcrumb', { trbRequestId: id }),
+            url: requestUrl
+          },
+          {
+            text: t('actionScheduleConsult.breadcrumb')
+          }
+        ]}
+      />
+
+      {message}
+
+      <Form
+        onSubmit={handleSubmit(formData => {
+          // Format the time as utc iso from the components' default local format
+          const consultMeetingTime = DateTime.fromFormat(
+            `${formData.meetingDate} ${formData.meetingTime}`,
+            'MM/dd/yyyy HH:mm'
+          )
+            .toUTC()
+            .toISO();
+
+          mutate({
+            variables: {
+              input: {
+                trbRequestId: id,
+                consultMeetingTime,
+                notes: formData.notes,
+                copyTrbMailbox: false,
+                notifyEuaIds: ['ABCD']
+              }
+            }
+          })
+            .then(result => {
+              showMessageOnNextPage(
+                <Alert type="success" slim className="margin-top-3">
+                  {t('actionScheduleConsult.success', {
+                    date: formData.meetingDate,
+                    time: DateTime.fromFormat(formData.meetingTime, 'HH:mm')
+                      .toFormat('t')
+                      .toLowerCase(),
+                    interpolation: {
+                      escapeValue: false
+                    }
+                  })}
+                </Alert>
+              );
+              history.push(requestUrl);
+            })
+            .catch(err => {
+              showMessage(
+                <Alert type="error" slim className="margin-top-3">
+                  {t('actionScheduleConsult.error')}
+                </Alert>
+              );
+            });
+        })}
+        className="maxw-full"
+      >
+        <Grid row>
+          <Grid col>
+            <PageHeading className="margin-bottom-0">
+              {t('actionScheduleConsult.heading')}
+            </PageHeading>
+            <div className="line-height-body-5 font-body-lg text-light">
+              {t('actionScheduleConsult.description')}
+            </div>
+            <div className="margin-top-1 margin-bottom-6 text-base">
```

Comment:
Bottom margin is a little big here - looks like `margin-bottom-4` would match the Figma.